joomla/joomla-cms
# Fixed: Consistency of "unpublished" -i con colours #43375

Fixed the inconsistency of the unpublished icon colour of the articles in the associations modal window.

Pull Request for Issue #43375 .

### Summary of Changes
Changed the unpublished icon from _icon-times_ (red) to _icon-unpublish_ (grey) in the article associations modal window.


### Testing Instructions

1. Make sure the multi-language feature is enabled by enabling the **System - Language Filter** plugin.
2. Install a second language if you have only one language installed.
3. Create an article for each language, set their state to **unpublished**.
4. Edit one of the two articles, go to the **Associations** tab and click the **Select** button to select an article.
5. Make sure the unpublished icon is grey.



### Actual result BEFORE applying this Pull Request
The unpublished icon color is red.
![unpublished icon is red](https://github.com/joomla/joomla-cms/assets/10974279/e9b67a97-166b-4e1c-80b2-45f5423b68a6)




### Expected result AFTER applying this Pull Request
The unpublished icon is grey.
![unpublished icon is grey](https://github.com/joomla/joomla-cms/assets/10974279/cec2135c-ef44-4ad1-ab41-3629f160b779)
